src/components/Notifications/NotificationTypes/Events/EventStartsSoonNotification.i18n.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react"
+import WatchLaterOutlinedIcon from "@mui/icons-material/WatchLaterOutlined"
+import { Link } from "@mui/material"
+import { NotificationItemCountdown } from "../../NotificationItem.styled"
+import { EventsStartsSoonNotificationProps } from "../../types"
+
+let interval: NodeJS.Timeout
+
+const Countdown = React.memo(({ startDate }: { startDate: string }) => {
```

Comment:
`Countdown` is a functional component, should it live here in the translations file?